library/src/main/java/com/schibsted/spain/barista/util/BitmapComparator.kt

```diff
@@ -0,0 +1,17 @@
+package com.schibsted.spain.barista.util
+
+import android.graphics.Bitmap
+import java.util.Arrays
+
+object BitmapComparator {
```

Comment:
Why not? I mean, this class doesn't need to be a class because it doesn't store any state and the method it has is a static method.

Reply:
Good news for you! 🎉  finally I had to change from object to class with just a companion object because the generated bytecode by Kotlin violated one findbugs rule 😞

Reply:
😂 
I'm just thinking, if this was Java what would we do? Doing this makes the class a singleton, and I wouldn't do that in Java. Why doing it in Kotlin then?

Reply:
Right! I didn't see it in that way...